README.md

```diff
@@ -43,41 +44,130 @@ Check the [Troubleshooting section](#troubleshooting) if something goes wrong wi
+
+* A Linux machine to run this tool (macOS may fail, see the [Troubleshooting section](#troubleshooting) in case you have problems running the tool).
 
-* GCP service account and credentials in the form of the environment variable
-`GOOGLE_APPLICATION_CREDENTIALS` with the path of the json credentials file for
-your service account (see [here](https://cloud.google.com/iam/docs/service-accounts) for details and make sure the service account has sufficient privileges for GKE).
 
 * Cloudflare credentials as two environment variables `CLOUDFLARE_EMAIL` and
 `CLOUDFLARE_API_KEY` (see [here](https://api.cloudflare.com/#getting-started)
 for details about the API key, the email hould be the one used for registration. Also, your domain name registra should be Cloudflare since this tool relies on Cloudflare for generating SSL certification).
 
-* Enough quota on GCP to create the required resources (terraform will show the
-exact errors if this condition is not met).
 
-* Kubernetes Engine API and billing enabled for your project, see [here](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+* GCP service account and credentials in the form of the environment variable
+`GOOGLE_APPLICATION_CREDENTIALS` with the path of the json credentials file for your service account (see [here](https://cloud.google.com/iam/docs/service-accounts)).
+The GCP configuration is required for use by the process to keep the built state. 
+
+* A project on GCP.
+
+* Configure specific requirements that depend on your infrastructure provider. More details on this subject are described on the following section for each of the specific providers.
+
 * Read through the [usage](#troubleshooting) section.
 
-The required steps to successfully deploy polkadot validator on GCP are as follows:
 
+---
+**NOTE**
+
+Running the following configurations will cause charges by the providers. You should run the corresponding destroy command as soon as you are finished with your testing to avoid unwanted expenses.
+
+---
+
+The required steps to successfully deploy polkadot validator are as follows:
+
+
+Download the latest polkadot deployer from git issuing the following command:  
+```git clone git@github.com:w3f/polkadot-deployer.git``` and run ```npm install``` to install all requirements.
+
+
+<details><summary>GCP</summary>
+<p>
+To make a deployment on GCP you are required to have the aforementioned GCP service account and project properly configured and meet the following requirements:
+
+* Make sure the service account has sufficient privileges for GKE.
+
+* Enough quota on GCP to create the required resources (terraform will show the exact errors if this condition is not met).
+
+* Kubernetes Engine API and billing enabled for your project (see [here](https://cloud.google.com/kubernetes-engine/docs/quickstart)).
+
+In order to deploy polkadot on GCP you can use a the preset configuration file: ```create.remote.sample-GCP.json``` and issue the following command:  
```

Comment:
I think this is misleading. The configuration is not going to be directly usable, you need to change the domain and the projectID at least. As I see it it is better to not proposing any example, just explaining how to configure each provider (env vars and required fields). Then at the end add a link to the relevant sections in `config/create.remote.sample.json`

Reply:
updated